src/main/java/de/variantsync/functjonal/Result.java

```diff
@@ -84,67 +88,76 @@ protected Result(final SuccessType result, final FailureType failure) {
-     * @param f Computation to run that indicates success with a boolean return value.
+     *
+     * @param f       Computation to run that indicates success with a boolean return value.
      * @param failure Factory for failure message in case f returned false.
      * @return Success iff f returned true, Failure otherwise.
      */
-    public static <F> Result<Unit, F> FromFlag(final Supplier<Boolean> f, final Supplier<F> failure) {
+    public static <F> Result<Unit, F> fromFlag(final Supplier<Boolean> f, final Supplier<F> failure) {
         if (f.get()) {
-            return Success(Unit.Instance());
+            return success(Unit.instance());
         } else {
-            return Failure(failure.get());
+            return failure(failure.get());
         }
     }
 
     /**
      * Runs the given computation that indicates success by returning a boolean and that may throw an exception.
      * Running f will be interpreted as a success iff f returns true and throws no exception.
      * Running f will be interpreted as a failure iff f returns false or throws an exception.
-     * If f did not throw an exception but returned false, a failure value will be produced with the given failure supplier.
-     * @param f Computation to run that indicates success with a boolean return value or an exception.
+     * If f did not throw an exception but returned false,
+     * a failure value will be produced with the given failure supplier.
+     *
+     * @param f       Computation to run that indicates success with a boolean return value or an exception.
      * @param failure Factory for failure message in case f returned false.
      * @return Success iff f returned true and did not throw an exception, Failure otherwise.
      */
-    public static <E extends Exception> Result<Unit, E> FromFlag(final FragileSupplier<Boolean, E> f, final Supplier<E> failure) {
+    public static <E extends Exception> Result<Unit, E> fromFlag(
+            final FragileSupplier<Boolean, E> f,
+            final Supplier<E> failure
+    ) {
         return Try(f).bibind(
                 Functjonal.when(
-                        () -> Success(Unit.Instance()),
-                        () -> Failure(failure.get())
+                        () -> success(Unit.instance()),
+                        () -> failure(failure.get())
                 ),
-                Result::Failure
+                Result::failure
         );
     }
 
     /**
      * Runs the given computation that may throw an exception.
-     * @param s Computation to run.
+     *
+     * @param s   Computation to run.
      * @param <E> The type of exception that may be thrown by s.
      * @return A result containing the result of the given computation or the exception in case it was thrown.
      */
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"unchecked", "checkstyle:TodoComment", "checkstyle:MethodName"})
```

Comment:
Ich finde den Methoden-Namen `try` echt gut. Leider ist das ja in keyword in java und mir ist keine gute Alternative eingefallen 🤔